### ISSUE TITLE
No module named 'PIL'

### ISSUE THREAD

**Issue by Hiisi5** [iodide-project/pyodide#800]
Hello. Using docker-image I was able to compile Pillow (PIL). I got Pillow.js and .data files which are accessible via localhost URL. pyodide.loadPackage is able to load .js file and diaplaying the contents of the virtual filesystem I can see the folder named PIL under 
```
pyodide._module.FS.readdir('/lib/python3.8/site-packages/'):
[".", "..", "PIL", "Pillow-8.0.1-py3.8.egg-info"]
```
However when I try `pyodide.runPython('import PIL')` it produses the error:
`ModuleNotFoundError: No module named 'PIL'`
How do I fix it? Did anybody yet build PIL? Because i used some dirty fixes in the process.

**Comment by Hiisi5** [comment]
Somehow pyodide uses python3.7. If I append sys.path with the path to 3.8 packages I am able to import PIL. Yet I have multiple issues with my build. So, did anyone build it yet and could possibly share? Thanks.

**Comment by rth** [comment]
Hi @Hiisi5,

> Did anybody yet build PIL? 

There was an attempt in https://github.com/iodide-project/pyodide/pull/733 but it's still WIP

> Somehow pyodide uses python3.7. If I append sys.path with the path to 3.8 packages I am able to import PIL

Pyodide v0.15.0 included Python 3.7 while the development version (and the next release) includes Python3.8. If you use the CDN download URLs replace `v0.15.0` by `dev` in the URL to get Python 3.8.
